File: dominio/modelos orijinales/QuitMenu.java
package servicebackmenu2;

abstract class QuitMenu extends NavigatorMenu {

    private QuitOption quitOption;

    public QuitMenu(String title) {
        super(title);
        this.quitOption = new QuitOption();
    }

    protected void showTitles() {
        this.add(this.quitOption);
        super.showTitles();
    }

}

File: conect4/src/main/resources/static/js/memus/models/generics/QuitMenu.js
import NavigatorMenu from "./NavigatorMenu.js";
import QuitOption from "./QuitOption.js";
class QuitMenu extends NavigatorMenu {
    
    #quitOption;

    constructor() {
        super();
        this.quitOption = new QuitOption();
    }

    addOptions() {
        super.addOptions();
       // this.addLast(this.quitOption);
    }

    addLast() {
        this.options.push(this.quitOption);
    }

    
}

export default QuitMenu;